### ISSUE TITLE
Save location sometimes fails 

### ISSUE THREAD

**Issue by Santobert** [home-assistant/home-assistant-polymer#3793]
**Home Assistant release with the issue:** 0.99.2

**Last working Home Assistant release (if known):** Not known

**UI (States or Lovelace UI?):** Lovelace

**Browser and Operating System:** Linux Firefox

**Description of problem:**
The map in general settings view sometimes failes to save the location. This is a frontend issue because there are wrong values for the longitude provided. This was already an issue in the backend repository, but nobody responded there and I can't fix it myself, so I'm creating another issue here. Unfortunately I can't close the old one.
The error occures only after you have tried to set a position when the map is scaled down to the minimum. After that it is no longer possible to set any other location.

Closes: home-assistant/home-assistant#26057

Steps to reproduce:
1. Scroll as far away as possible
2. Grab and move the map with your mouse
3. Select a location (Map is still scaled down)
4. Hit the save button
5. Repeat if no error occurs

**Javascript errors shown in the web inspector (if applicable):** None

**Python errors shown in the command line (if applicable):**
```
2019-09-23 07:18:18 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection.139726041231888] Error handling message: invalid longitude for dictionary value @ data['longitude']. Got 205.31250000000003
Traceback (most recent call last):
  File "/home/tobi/workspace/home-assistant/homeassistant/components/websocket_api/connection.py", line 91, in async_handle
    handler(self.hass, self, schema(msg))
  File "/home/tobi/workspace/home-assistant/venv/lib/python3.7/site-packages/voluptuous/schema_builder.py", line 272, in __call__
    return self._compiled([], data)
  File "/home/tobi/workspace/home-assistant/venv/lib/python3.7/site-packages/voluptuous/schema_builder.py", line 594, in validate_dict
    return base_validate(path, iteritems(data), out)
  File "/home/tobi/workspace/home-assistant/venv/lib/python3.7/site-packages/voluptuous/schema_builder.py", line 432, in validate_mapping
    raise er.MultipleInvalid(errors)
voluptuous.error.MultipleInvalid: invalid longitude for dictionary value @ data['longitude']
```

**Additional information:**
